pkg/pillar/dpcreconciler/linuxitems/adapter.go

```diff
@@ -168,6 +161,15 @@ func (c *AdapterConfigurator) Create(ctx context.Context, item depgraph.Item) er
 		c.Log.Error(err)
 		return err
 	}
+	// Make sure that the interface is UP right after MAC address assignment
+	// and interface rename. In case of incorrect order operation fails
+	// with `Device or resource busy`.
+	if err := netlink.LinkSetUp(link); err != nil {
```

Comment:
I suggest to put this at the end (after `LinkSetUp(bridge)`) and with `kernLink` as the argument. In the past I saw `LinkSetMaster` changing the interface status to DOWN.

Reply:
Done.